server/plugin.go

```diff
@@ -232,3 +233,15 @@ func (p *Plugin) MessageHasBeenPosted(c *plugin.Context, post *model.Post) {
 		return
 	}
 }
+
+func (p *Plugin) FileWillBeUploaded(c *plugin.Context, info *model.FileInfo, file io.Reader, output io.Writer) (*model.FileInfo, string) {
```

Comment:
We should add a configuration setting for this. 
It also needs to follow the security restrictions.

Reply:
We may also want to devise an easy way to skip this in case of already transcribed files (e.g. calls transcriptions).